File: python/1_sphere_volume.py
# sphere_volume(R) : a function that computes the volume of a sphere, given
# its radius R.

import math

def sphere_volume(R):
    return (4/3) * math.pi * R**3

# Example usage:
print(sphere_volume(1))  # 4.1887902047863905
print(sphere_volume(2))  # 33.510321638291124

File: julia/1_sphere_volume.jl
# sphere_volume(r): a function that computes the volume of a sphere,
# given its radius r.

function sphere_volume(r)
    return (4/3) * pi * r^3
end

# Example usage:
radius = 5.0
volume = sphere_volume(radius)
println("The volume of a sphere with radius $radius is $volume")
# The volume of a sphere with radius 5.0 is 523.5987755982989
